src/main/kotlin/persistence/mapping/UserMapper.kt

```diff
@@ -0,0 +1,61 @@
+package persistence.mapping
+
+import data.model.Language
+import data.model.User
+import data.dao.Dao
+import data.mapping.Mapper
+import persistence.repo.UserLanguageRepo
+import persistence.tables.daos.UserPreferencesEntityDao
+import persistence.tables.pojos.UserEntity
+
+class UserMapper(
+        private val userLanguageRepo: UserLanguageRepo,
+        private val languageRepo: Dao<Language>,
+        private val userPreferencesEntityDao: UserPreferencesEntityDao
+): Mapper<UserEntity, User> {
```

Comment:
Going to switch from using our repos to the jooq generated Dao, so that way we depend to directly on jooq's generated code, rather than our repos which are subject to change

Reply:
Did the change in the jooq branch, repos no longer depend on each other an just on generated daos